.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ executors:
 parameters:
     current_golden_images_hash:
         type: string
-        default: 10500edb351f4db403b24e1305a959910ef80e12
+        default: ad6a2c7402744826239beac9f89e022a588da9f0
```

Comment:
The CI failures point to what I think is a more expected _lack_ of changs from `main`. E.G. when I parse changes in the `*medium-vars.css` files there do not seem to be any changes for "checkbox", but I 'd love you insight on what this update was _supposed_ to do.

The previously captured changes may have been part of the dirty `yarn.lock`? 😬